examples/index.html

```diff
@@ -48,7 +48,6 @@ <h3 class="font-size-small">In Text:</h3><br/>
   <script>
     var blurredLocation = new BlurredLocation();
```

Comment:
Lets show how to set interfaceOptions in here -- you can add comments saying `} // optional` but it's one reason the example is nice to have -- to show how it's all used.